app/views/admin/attachments/_form.html.erb

```diff
@@ -23,23 +23,39 @@
 
   <% if attachment.is_a?(HtmlAttachment) %>
     <%= form.fields_for :govspeak_content do |govspeak_fields| %>
+      <div class="govuk-!-margin-bottom-8">
+        <%= render "govuk_publishing_components/components/checkboxes", {
+          name: "attachment[govspeak_content_attributes][manually_numbered_headings]",
+          items: [
+            {
+              label: "Manually numbered headings",
+              value: "1",
+              checked: form.object.content_id != nil
+            }
+          ]
+        } %>
 
-      <%= govspeak_fields.check_box :manually_numbered_headings %>
-      <div class="js-manual-numbering-help<%= ' js-hidden' unless attachment.manually_numbered_headings? %>">
-        <pre>
-  ## 1. First heading
-  ### 1.1 First sub-heading
-  ### 1.2 Second sub-heading
-  ### Unnumbered sub-heading
-
-  Manually number your headings as appropriate using the above numbering scheme.</pre>
+        <%= render "govuk_publishing_components/components/details", {
+          title: "Guidance on manually numbered headings"
+        } do %>
+          <p>Manually number your headings using the numbering scheme below.</p>
+          <code>## 1. First heading<br />### 1.1 First sub-heading<br />### 1.2 Second sub-heading<br />### Unnumbered sub-heading</code>
+        <% end %>
       </div>
 
-      <%= content_tag :fieldset, class: ("right-to-left" if form.object.rtl_locale?) do %>
-        <%= govspeak_fields.text_area :body, rows: 20, class: "previewable", data: {
-          module: "paste-html-to-govspeak"
+      <div class="govuk-!-margin-bottom-8">
```

Comment:
you can add the margin bottom flag on the text area: 

```
margin_bottom: 9,
```

Reply:
https://components.publishing.service.gov.uk/component-guide/textarea/with_margin_bottom

Reply:
👍